docs/current_docs/quickstart/daggerize.mdx

```diff
@@ -25,6 +25,10 @@ The example application is a skeleton Vue framework application that returns a "
 git clone https://github.com/dagger/hello-dagger
 cd hello-dagger
 ```
+:::tip Develop
+1. By default the `internal` directory and the `dagger.gen.go` are created during `dagger init` are added to a `.gitignore` in your dagger directory
```

Comment:
Isn't this message confusing at this stage? Specially since the recently cloned hello-dagger repo doesn't have any internal or generated files yet.

Reply:
Good point. I'll reread and find a better spot.

Reply:
This could be added to https://docs.dagger.io/api/custom-functions#initialize-a-dagger-module